internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -313,7 +314,7 @@ func (opts *Opts) defaultValues() error {
 		if publicIP := store.IPAddress(); publicIP != "" {
 			opts.IPAddresses = []string{publicIP}
 		} else {
-			fmt.Print(quickstartTemplateIPNotFound)
+			return errors.New(quickstartTemplateIPNotFound)
```

Comment:
sorry if it wasn't clear before but this is what I meant 
```suggestion
			_, _ = fmt.Fprintln(os.Stderr, quickstartTemplateIPNotFound)
```

Reply:
I don't understand why using `Fprintln`here given that we do not have a formatted string (the string does not have %s). `Errors.new` prints to stderr anyway.

Reply:
sorry, I see what is the difference here only now 🤦  we have to use `fmt.Fprintln` so that the user can still use the command even if the endpoint does not find the public IP. I am going to update it. Thanks!